scarbine/arete
# user can now add gear to their gear list

#### Changes Made


1. Modified file `GearCard.js` to include `addClimberGear()` and functionality.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout adding-to-climbers-gear-list
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
   > Example: When user hits the affordance to add gear, it should be added to their list of gear.
4. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
